lib/msf/ui/console/driver.rb

```diff
@@ -313,8 +313,6 @@ def load_resource(path)
       return
     end
 
-    self.active_resource = resource_file
```

Comment:
I found nothing that used this anywhere, even in pro, so deleted to make it possible to make this a static method someday.

Reply:
I did a quick search as well and didn't see the property being used.